corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -165,7 +156,23 @@ def _testUpdate(self, sync_log_or_id, case_id_map, dependent_case_id_map=None):
             self._testUpdate(migrated_sync_log, case_id_map, dependent_case_id_map)
 
 
-class SyncTokenUpdateTest(SyncBaseTest):
+class SyncBaseTest(BaseSyncTest):
```

Comment:
Maybe change this name to `DeprecatedBaseSyncTest`? (Right now the only difference in their names is transposing Sync and Base, which doesn't really tell you anything about which is preferred.)